Doc/reference/compound_stmts.rst

```diff
@@ -1057,7 +1057,7 @@ subject value:
 
       * If this raises an exception, the exception bubbles up.
 
-      * If the returned value is not a list or tuple, the conversion fails and
```

Comment:
There's some discussion on putting this check when the attribute is set rather than looked up. I wouldn't change this until this is decided

Reply:
Yeah I agree. I'm waiting for the other two PRs in the PEP repo to get finalised and merged. Would be nice if someone added the ``DO-NOT-MERGE`` label to this PR temporarily (I'm unable to do so).

Reply:
> Would be nice if someone added the `DO-NOT-MERGE` label to this PR temporarily (I'm unable to do so).

I turned it into draft mode instead (you should be able to un-draft it yourself -- let me know if you can't).

Reply:
Well I'm feeling a little silly now - I forgot about drafts. Thanks Guido! (I'm also able to un-draft it myself)

Reply:
> There's some discussion on putting this check when the attribute is set rather than looked up. I wouldn't change this until this is decided

I don't think we'll change when the check is done right away, but we're changing the acceptable type to tuple right away (#25203), so the docs need to be changed. At what point the check is done would affect the tiniest fraction of all code, but disallowing lists might already affect existing (bleeding edge) code.